Makefile

```
@@ -135,6 +135,8 @@ controller.yaml: controller.jsonnet controller-norbac.jsonnet
 
 controller-norbac.yaml: controller-norbac.jsonnet
 
+controller-podmonitor.yaml: controller.jsonnet controller.image controller-norbac.jsonnet
```

as of #367  (multi-arch containers) the *.yaml rules no longer depend on the controller.image

```suggestion
controller-podmonitor.yaml: controller.jsonnet controller-norbac.jsonnet
```

---

Ah, missed this in rebase, done now.